### DISCUSSION TITLE
It's possible to convert to png after cropping ?

### DISCUSSION THREAD

**Discussion by anarkrypto** [ValentinH/react-easy-crop#452] [General]
Since it supports jpeg and png, it's possible to export cropped images directly to png always ?

**Comment by ValentinH** [comment]
Yes it's possible.

If you check this example: https://codesandbox.io/embed/q8q1mnr01w

In the crop image.js file, you can replace jpeg by png at line 90. Then, you only have to adapt the example to save the image instead of displaying it inside a popup.
